documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

```diff
@@ -6,13 +6,13 @@
 = Strimzi custom resource upgrades
 
 [role="_abstract"]
-After you have upgraded Strimzi to {ProductVersion}, you must ensure that your custom resources are using API version `{ApiVersion}`.
-You can do this any time after upgrading to {ProductVersion}, but the upgrades must be completed before the next Strimzi minor version update.
+Before upgrading Strimzi to {ProductVersion}, you must ensure that your custom resources are using API version `{ApiVersion}`.
+You can do this any time after upgrading to Strimzi {PreviousProductVersion}, but the upgrades must be completed before upgrading to {ProductVersion}.
```

Comment:
It looks like this was wrong already before. But this should probably use some special variables or just have the versions in the text. The CRD upgrade is needed:
* After upgrading to 0.22
* Before upgrading to 0.23 or newer

The way it is now here is not correct since it will render as _After 0.23 and before 0.24_ + change with every release.

Reply:
I've created a couple of variables: 

:ConvertAfterProductVersion: 0.22.0
:ConvertBeforeProductVersion: 0.23.0